build-a-react-native-application-for-ios-and-android-from-start-to-finish/notes/23-show-a-spinner-while-submitting-a-form-in-react-native-with-activityindicator.md

```diff
@@ -0,0 +1,61 @@
+# Show a Spinner while Submitting a Form in React Native with ActivityIndicator
+
+**[📹 Video](https://egghead.io/lessons/react-native-show-a-spinner-while-submitting-a-form-in-react-native-with-activityindicator)**
+
+## Loading spinners
+
+When loading data from an API or another remote source, the request can take a significant amount of time. Loading spinners are a good way to communicate to the user that something is happening.
+
+The [`<ActivityIndicator />`](https://reactnative.dev/docs/activityindicator) can show a spinner while content is loading or submitting. It can be provided a `size` and `color` prop, however, managing whether it is shown or not is done manually with state.
```

Comment:
```suggestion
🤔  The [`<ActivityIndicator />`](https://reactnative.dev/docs/activityindicator) can show a spinner while content is loading or submitting. It can be provided a `size` and `color` prop, however, managing whether it is shown or not is done manually with state.
```